test/unit/helpers/importedFiles.test.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+const importedFiles = require('../../../src/helpers/importedFiles');
+
+jest.mock('@bem/sdk.cell',
+    () => ({ create: obj => obj })
+);
+
+jest.mock('../../../src/helpers/namingConvention',
+    () => ({ getFromLevel: () => '[origin]' })
+);
+
+jest.mock('../../../src/helpers/cellPath',
+    () => (cell, naming) => {
+        expect(naming).toBe('[origin]');
+
+        return `${cell.layer}.blocks/${cell.entity.block}/${cell.entity.block}.${cell.tech}`;
```

Comment:
`file.naming.stringify` doesnt work well?

Reply:
I also mocked getter of naming (see few lines before - naming is simple string) so true `bem/sdk.naming.*.stringify` will fail.

Plugin doesn't use `bem/sdk.*` directly only through "helpers". After https://github.com/bem/webpack-bem-plugin/issues/10 we will remove helpers and such mocks won't be needed.

On the other side mocking of all dependencies is good for unit testing. Whole thing is tested by e2e